### PR TITLE
fixed the bug I opened

### PR DESCRIPTION
**Pull request by trevorshoaf**

### CODE REVIEW

**Comment by trevorshoaf** [comment]
https://github.com/thomashirtz/notion-scholar/issues/12

File: notion_scholar/bibtex.py [hunk 1]
```diff
@@ -82,6 +82,7 @@ def get_publication_list(bib_database: BibDatabase) -> List[Publication]:
                 doi=entry.get('doi', ''),
                 type=entry.get('ENTRYTYPE', '').lower(),
                 bibtex=get_bibtex_str(entry),
+                keywords=entry.get('keywords',''),
```
Comment (thomashirtz) [inline notion_scholar/bibtex.py:85]: No need to add this line

File: notion_scholar/publication.py [hunk 2]
```diff
@@ -13,6 +13,7 @@ class Publication(NamedTuple):
     abstract: str
     doi: str
     type: str
+    keywords: str
```
Comment (thomashirtz) [inline notion_scholar/publication.py:16]: No need to add this line

File: notion_scholar/notion_api.py [hunk 3]
```diff
@@ -84,7 +84,6 @@ def add_publications_to_database(
                 'Authors': Property.rich_text(authors),
                 'Year': Property.number(publication.year),
                 'URL': Property.url(publication.url),
-                'Category': Property.relation(publication.keywords),
```
Comment (thomashirtz) [inline notion_scholar/notion_api.py:87]: You can do this PR with only this line removed and I would accept it

**Pull request closed without merging**